### ISSUE TITLE
Select & related compoents not accessible on ios with voiceover 

### ISSUE THREAD

**Issue by madebyjonny** [primefaces/primevue#8176]
### Describe the bug

When trying to interact with the select (& other similar components e.g. auto complete) while voiceover is enabled, the focus doesn't go to the drop down when swiping right as expected and there doesn't seem to be a way to get the focus onto it without pressing onto the ui itself. 

### Reason for not contributing a PR

- [x] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://primevue.org/select

### Environment

iOS 18.6.2

### Vue version

3.16.2 (nuxt)

### PrimeVue version

4.3.8

### Node version

22.14.0

### Browser(s)

Mobile Safari 18

### Steps to reproduce the behavior

have voiceover enabled on a mobile device 
navigate to a select ( i used the docs for this )
once the select is focused, double tap 
the focus remains on the page with swiping left/right 

### Expected behavior

I would expect the focus to go automatically to the drop down once swiping right after initial interaction

**Comment by madebyjonny** [comment]
Adding append-to="self" seems to get the focus to the drop down, but once selected the foucs needs to be handled
